web/ui/react-app/src/pages/graph/Graph.test.tsx

```diff
@@ -9,6 +9,19 @@ describe('Graph', () => {
   beforeAll(() => {
     jest.spyOn(window, 'requestAnimationFrame').mockImplementation((cb: any) => cb());
   });
+
+  // fix coming from https://github.com/maslianok/react-resize-detector#testing-with-enzyme-and-jest
+  beforeEach(() => {
+    window.ResizeObserver = jest.fn().mockImplementation(() => ({
+      observe: jest.fn(),
+      unobserve: jest.fn(),
+      disconnect: jest.fn(),
+    }));
+  });
+
+  afterEach(() => {
+    window.ResizeObserver = ResizeObserver;
+  });
```

Comment:
don't we need an extra blank line?

Reply:
mmm I think if it was the case, then the linter would yell

Reply:
This is just a place where a human would expect an empty line (not relevant to the linter though), so let's put one there.

Reply:
ah yeah ok, I misunderstood it was for code sanity. Yeah make sense.